index.html

```diff
@@ -0,0 +1,204 @@
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="tools">
+                    <li><label class="tool-press">HTML</label></li>
+                    <li><label class="tool-press">CSS</label></li>
+                    <li><label class="tool-press">JAVASCRIPT</label></li>
+                </ul>
+                <button class="see-project">See Project</button>
+            </div>
+        </div>
+        <!-- End of Project 1 -->
+
+        <!-- Start of Project 2 -->
+        <div id="tonic-project">
+            <div class="data">
+                <img src="./images/snapshots/Snapshoot Portfolio 2.png" alt="Tonic project snapshot">
+                <h2 class="project-title">Multi-Post <br />Stories</h2>
+                <div class="work-nature">
+                    <label class="canopy">CANOPY</label>
+                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                    <label class="backend"> Back End Dev</label>
+                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                    <label class="year">2015</label>
+                </div>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="tools">
+                    <li><label class="tool-press">HTML</label></li>
+                    <li><label class="tool-press">CSS</label></li>
+                    <li><label class="tool-press">JAVASCRIPT</label></li>
+                </ul>
+                <button class="see-project">See Project</button>
+            </div>
+        </div>
+        <!-- End of Project 2 -->
+
+        <!-- Start of Project 3 -->
+        <div id="tonic-project">
+            <div class="data">
+                <img src="./images/snapshots/Snapshoot Portfolio 3.png" alt="Tonic project snapshot">
+                <h2 class="project-title">Tonic</h2>
+                <div class="work-nature">
+                    <label class="canopy">CANOPY</label>
+                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                    <label class="backend"> Back End Dev</label>
+                    <div class="bullet-point"><img src="./images/icons/Counter.png" alt=""></div>
+                    <label class="year">2015</label>
+                </div>
+                <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul class="tools">
+                    <li><label class="tool-press">HTML</label></li>
+                    <li><label class="tool-press">CSS</label></li>
+                    <li><label class="tool-press">JAVASCRIPT</label></li>
+                </ul>
+                <button class="see-project">See Project</button>
```

Comment:
- [Optional] Consider adding a hover and pressed state to all your buttons as it is on the figma template. You can use the `:hover` and `:active` pseudoclasses for this.
![image](https://user-images.githubusercontent.com/94551484/227545089-c6ed521a-18ff-4bee-9e00-8f06897df79c.png)

Reply:
@Kamba56 Thank you very much for you input.